cfme/containers/project.py

```diff
@@ -36,6 +41,11 @@ class Project(BaseEntity, Taggable, Labelable):
     def mgmt(self):
         return ApiProject(self.provider.mgmt, self.name)
 
+    @classmethod
+    def get_project_entity(cls, provider, provider_name):
+        selected_provider = provider.mgmt.get_project_by_name(provider_name)
+        return cls(selected_provider.name, provider)
```

Comment:
**Required**

Remove this, its serving the same function as the `ProjectCollection.instantiate` method provided by `BaseCollection`.  Maybe there's some other purpose or need that `instantiate` isn't solving, but I don't think this is the correct way to instantiate project objects.  They will have no appliance attribute.

Also this function isn't used at all in this PR, what's the intended purpose?  The function should have a docblock that explains this, if it even is necessary.

Reply:
Sorry, I didn't pay attention to that.
This function is leftover from origin Pavel PR were there was no collection implementation yet.
Indeed I use the ProjectCollection.instantiate on my test instead.
I removed this function.